database/log/list.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2023 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package log
+
+import (
+	"github.com/go-vela/types/constants"
+	"github.com/go-vela/types/database"
+	"github.com/go-vela/types/library"
+)
+
+// ListLogs gets a list of all logs from the database.
+func (e *engine) ListLogs() ([]*library.Log, error) {
+	e.logger.Trace("listing all logs from the database")
+
+	// variables to store query results and return value
+	count := int64(0)
+	h := new([]database.Log)
```

Comment:
does `h` signify something or is that copied from hooks? 
not a big deal

Reply:
This has been addressed within https://github.com/go-vela/server/pull/780